src/swagger-specs/streaming-ingestion.yml

```diff
@@ -0,0 +1,405 @@
+          content: {}
+      x-codegen-request-body-name: body
+components:
+  schemas:
+    streamingMessage:
+      required:
+      - DATASET_ID
+      - IMS_ORG_ID
+      - SCHEMA_ID
+      - SCHEMA_VERSION
+      type: object
+      properties:
+        TENANT_ID:
+          type: string
+          description: The tenant ID ensures that the resources you create are properly
+            namespaced and contained within your organization.
+        SCHEMA_ID:
+          type: string
+          description: The ID of the schema that is created.
+        SCHEMA_VERSION:
+          type: string
+          description: The version of the schema that is created.
+        IMS_ORG_ID:
+          type: string
+          description: Your organization ID can be found under the integration
+            details in the Adobe Developer Console.
+        DATASET_ID:
+          type: string
+          description: The ID of the dataset that you want to publish to.
+        SOURCE_NAME:
+          type: string
+          description: An optional name for your source. If this value is missing,
+            the streaming message will automatically add the source ID from the streaming
+            connection definition.
+      description: The message that you want to ingest.
+      example:
+        header:
+          schemaRef:
+            id: https://ns.adobe.com/{TENANT_ID}/schemas/{SCHEMA_ID}
+            contentType: application/vnd.adobe.xed-full+json;version={SCHEMA_VERSION}
+          imsOrgId: '{IMS_ORG_ID}'
+          datasetId: '{DATASET_ID}'
+          createdAt: "1526283801869"
+          source:
+            name: '{SOURCE_NAME}'
+        body:
+          xdmMeta:
+            schemaRef:
+              id: https://ns.adobe.com/{TENANT_ID}/schemas/{SCHEMA_ID}
+              contentType: application/vnd.adobe.xed-full+json;version={SCHEMA_VERSION}
+          xdmEntity:
+            person:
+              name:
+                firstName: Jane
+                middleName: F
+                lastName: Doe
+              birthDate: 1969-03-14
+              gender: female
+            workEmail:
+              primary: true
+              address: janedoe@example.com
+              type: work
+              status: active
+    StreamingIngestionResponse:
+      type: object
+      properties:
+        inletID:
+          type: string
+          description: The ID of the previously created streaming endpoint.
+        xactionId:
+          type: string
+          description: The `xactionID` is a unique, server-side generated identifier 
+            for the XDM record that you just sent. This ID helps with debugging and 
+            for Adobe to trace this record's lifecycle through it's various systems.
+        receivedTimeMs:
+          type: integer
+          description: The timestamp in **milliseconds** that shows the time at which
+            the request was received.
+          format: int32
+      example:
+        inletId: '{INLET_ID}'
```

Comment:
Optional but if you want a concrete example for Inlet ID, you can use this one that I made: `667b41cf2dbf3509927da1ebf7e93c20afa727cc8d8373e51da18b62e1b985ec`